Tests/SwiftParserTest/AttributeTests.swift

```diff
@@ -1283,6 +1283,49 @@ final class AttributeTests: ParserTestCase {
     )
   }
 
+  func testABIAttributeWithoutFeature() {
+    assertParse(
+      """
+      @abi(1️⃣func fn() -> Int2️⃣)
+      func fn1() -> Int { }
+      """,
+      substructure: FunctionDeclSyntax(
+        attributes: [
+          .attribute(
+            AttributeSyntax(
+              attributeName: TypeSyntax("abi"),
+              leftParen: .leftParenToken(),
+              [Syntax(try! FunctionDeclSyntax("func fn() -> Int"))],
```

Comment:
Could make the test function `throws` and then avoid the `try!` here.
 
```suggestion
              [Syntax(try FunctionDeclSyntax("func fn() -> Int"))],
```